src/python/pants/notes/1.11.x.rst

```diff
@@ -0,0 +1,257 @@
+1.11.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.11.x`` ``stable`` series.
+
+1.11.0rc0 (10/16/2018)
```

Comment:
The `1.11.0rc0` section should only contain the changes since the most recent `1.11.0` dev release, and then the rest of the `1.11.0` `dev` releases should be listed on this page.

Reply:
See the other notes docs for more info.